manifest.yaml

```diff
@@ -66,35 +58,10 @@ spec:
       maxSurge: 25%
 ---
 apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: nginx-conf
-  namespace: activemq-artemis-self-provisioning-plugin
-  labels:
-    app: activemq-artemis-self-provisioning-plugin
-    app.kubernetes.io/part-of: activemq-artemis-self-provisioning-plugin
-data:
-  nginx.conf: |
-    error_log /dev/stdout info;
-    events {}
-    http {
-      access_log         /dev/stdout;
-      include            /etc/nginx/mime.types;
-      default_type       application/octet-stream;
-      keepalive_timeout  65;
-      server {
-        listen              9443 ssl;
-        ssl_certificate     /var/serving-cert/tls.crt;
-        ssl_certificate_key /var/serving-cert/tls.key;
-        root                /usr/share/nginx/html;
-      }
-    }
----
-apiVersion: v1
 kind: Service
 metadata:
   annotations:
-    service.alpha.openshift.io/serving-cert-secret-name: plugin-serving-cert
+    service.beta.openshift.io/serving-cert-secret-name: plugin-serving-cert
```

Comment:
Are we eliminating compatibility with any older versions by moving from alpha to beta here? If so it's a good item to to point out via the README.md and other docs.